### PR TITLE
Add ssh key for accessing workers from galaxy-backup, galaxy, galaxy-queue

### PR DESCRIPTION
**Pull request by cat-bro**
From galaxy-queue we could ssh to a worker with 'ssh galaxy-w10' for example. galaxy-w10 will only accept this ssh key from galaxy, galaxy-queue or galaxy-backup.

This does not allow every galaxy machine access to every other one - nothing can use this key to get to the galaxy VM and we should probably allow access from galaxy-backup, in which case this could actually replace galaxy-australia as the deployment key.

### CODE REVIEW

**Review by jlqfab — DISMISSED** [review]
does galaxy-handlers need to be added somewhere?

**Comment by cat-bro** [comment]
this isn't ready yet, just rebased

**Comment by cat-bro** [comment]
@jlqfab this is ready for review

File: group_vars/galaxy_workers.yml [hunk 1]
```diff
@@ -28,3 +28,9 @@ docker_users:
   - "{{ galaxy_user.name }}"
 docker_daemon_options: # TODO: Uncomment as soon as there is a volume
   data-root: /mnt/docker-data
+
+# internal ssh keys
+extra_keys:
+  - id: internal_hop_key
+    type: public
+    from: "{{ hostvars['galaxy']['internal_ip'] }},{{ hostvars['galaxy-backup']['internal_ip'] }},{{ hostvars['galaxy-queue']['internal_ip'] }}"
```
Comment (jlqfab) [inline group_vars/galaxy_workers.yml:36]: should we also add galaxy-handlers internal_ip here?

Comment (cat-bro) [reply group_vars/galaxy_workers.yml]: yes. pushing change now.

File: roles/common/tasks/main.yml [hunk 2]
```diff
@@ -91,6 +90,14 @@
   loop_control:
     loop_var: extra_key
 
+- name: Add ssh config
+  import_tasks: ssh_config.yml
+  when: ssh_config_hosts is defined
+
+- name: Add ssh config
```
Comment (jlqfab) [inline roles/common/tasks/main.yml:97]: this duplicates the above task

File: host_vars/galaxy-handlers.usegalaxy.org.au.yml
Comment (jlqfab) [inline host_vars/galaxy-handlers.usegalaxy.org.au.yml:1]: I think this also needs the following defined, as per galaxy, galaxy-backup, galaxy-queue
```
# ssh config, only for ubuntu
ssh_config_id_file: "/home/{{ ssh_config_user }}/.ssh/internal_hop_key"
ssh_config_user: ubuntu
ssh_config_hosts: "{{ galaxy_group }}"
```

File: roles/common/tasks/main.yml (hunk 2 above)
Comment (cat-bro) [reply roles/common/tasks/main.yml]: oops. new commit removes the second copy

File: host_vars/galaxy-handlers.usegalaxy.org.au.yml
Comment (cat-bro) [reply host_vars/galaxy-handlers.usegalaxy.org.au.yml:1]: it will actually inherit that from galaxy.usegalaxy.org.au

**Review by jlqfab — APPROVED** [review]

**Pull request merged**